TransformationSystem/Agent/TransformationAgent.py

```diff
@@ -72,7 +72,8 @@ def initialize( self ):
     self.pluginLocation = self.am_getOption( 'PluginLocation',
                                              'DIRAC.TransformationSystem.Agent.TransformationPlugin' )
     self.transformationStatus = self.am_getOption( 'transformationStatus', ['Active', 'Completing', 'Flush'] )
-    self.maxFiles = self.am_getOption( 'MaxFiles', 5000 )
+    # Prepare to change the name of the CS option as MaxFiles is ambiguous
+    self.maxFiles = self.am_getOption( 'MaxFilesToProcess', self.am_getOption( 'MaxFiles', 5000 ) )
```

Comment:
Please update doc at http://dirac.readthedocs.io/en/rel-v6r17/AdministratorGuide/Configuration/ConfReference/Systems/Transformation/Agents/transformationagent.html?highlight=MaxFiles (in this PR)